entity_plus.module

```diff
@@ -1094,3 +1094,47 @@ function _entity_plus_modules_get_default_types($modules) {
   }
   return $types;
 }
+
+/**
+ * Implements hook_modules_disabled().
+ */
+function entity_plus_modules_disabled($modules) {
+  foreach (_entity_plus_modules_get_default_types($modules) as $entity_type) {
+    $info = entity_get_info($entity_type);
+
+    // Do nothing if the module providing the entity type has been disabled too.
+    if (isset($info['module']) && in_array($info['module'], $modules)) {
+      return;
+    }
+
+    $keys = $info['entity keys'] + array('module' => 'module', 'status' => 'status', 'name' => $info['entity keys']['id']);
+    // Remove entities provided in code by one of the disabled modules.
+    $query = new EntityFieldQuery();
+    $query->entityCondition('entity_type', $entity_type, '=')
+          ->propertyCondition($keys['module'], $modules, 'IN')
+          ->propertyCondition($keys['status'], array(ENTITY_PLUS_IN_CODE, ENTITY_PLUS_FIXED), 'IN');
+    $result = $query->execute();
+    if (isset($result[$entity_type])) {
+      $entities = entity_load($entity_type, array_keys($result[$entity_type]));
```

Comment:
Here, please use `entity_load_multiple()` instead of `entity_load`. While `entity_load` CAN return an array (backward compatibility), it's marked as an error by most IDE. On the other hand `entity_load_multiple()` returns an array unequivocally.